stdlib/src/utils/static_tuple.mojo

```diff
@@ -126,14 +126,14 @@ struct StaticTuple[element_type: AnyRegType, size: Int](Sized):
     """The underlying storage for the static tuple."""
 
     @always_inline
-    fn __init__() -> Self:
+    fn __init__(inout self):
         """Constructs an empty (undefined) tuple.
 
         Returns:
             The tuple.
         """
         _static_tuple_construction_checks[size]()
-        return Self {array: __mlir_op.`kgen.undef`[_type = Self.type]()}
+        self = Self {array: __mlir_op.`kgen.undef`[_type = Self.type]()}
```

Comment:
I think this should be:
self.array = __mlir_op.`kgen.undef`[_type = Self.type]()

Reply:
I wasn't sure if the behavior of `kgen.undef` will be preserved outside of brackets. Thank you, updated.